build/common.props

```diff
@@ -8,10 +8,25 @@
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)Key.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
```

Comment:
...oh, I remember why I did this. NETStandard.Library is not implied on .NET Framework tfms.